apertium-crh.crh.lexc

```diff
@@ -18,107 +23,119 @@ Multichar_Symbols
 %<cnjcoo%>        ! Coordinating conjunction    !
-%<cnjsub%>        ! "subordinating conjunction" (ki)
-%<cnjadv%>        ! 
-%<qst%>           ! Question word
-%<sym%>
+%<cnjsub%>        ! Subordinating conjunction (ki)
+%<cnjadv%>        ! Adverbial conjunction
+%<qst%>           ! Modal question particle     !# м{I}
+                  
+%<sym%>           ! Symbol
 %<cop%>           ! Copula
-%<postadv%>       ! 
-%<det%>           ! 
-%<num%>           ! 
+%<postadv%>       ! Post of Adverb
+%<det%>           ! Determiner
+%<num%>           ! Numeral
 %<vaux%>          ! Auxiliary verb
 %<abbr%>          ! Abbreviation
-%<emph%> 
-%<ij%>
-%<nom%>           ! Nominative                  !
-%<gen%>           !
-%<dat%>           !
-%<acc%>           !
-%<loc%>           ! Locative                    ! 
-%<abl%>           ! Ablative                    !
-%<ins%>           !
-%<sim%>
-
-%<iv%>            !
-%<tv%>            !
-
-%<nec%>
-%<imp%>
-%<aor%>
-%<gno%>
-%<fut%>
+%<emph%>          ! Emphasizing modal particle  !# -ч{I}, -с{A}н{A}, әле
+%<ij%>            ! Interjection
+%<nom%>           ! Nominative
+%<gen%>           ! Genitive
+%<dat%>           ! Dative
+%<acc%>           ! Accusative
+%<loc%>           ! Locative
+%<abl%>           ! Ablative
+%<ins%>           ! Instrumental=comitative
+%<sim%>           ! Similative
+
+! Transitivity
+%<iv%>            ! Transitive
+%<tv%>            ! Intransitive
+
+%<nec%>           ! Necessitive
+%<imp%>           ! Imperative
+%<aor%>           ! Aorist
+%<gno%>           ! Gnomic tense
+%<fut%>           ! -{I}р
 %<fut2%>          ! -EcAG
-%<past%>
-%<ifi%>
 
-%<neg%>
+%<neg%>           ! Negative
 
-%<pass%>
-%<coop%>
-%<caus%>
-%<abil%>
-%<evid%>
+%<pass%>          ! Passive
+%<coop%>          ! Cooperative
+%<caus%>          ! Causative
+%<abil%>          ! Abilative
+%<evid%>          ! Evidential
 
-%<ger%>
-%<ger_pres%>
-%<ger_fut2%>
-%<ger_past%>
+! Gerunds (verbal nouns)
+%<ger%>           ! -m{A}{G}
+%<ger_pres%>      ! -{j}{A}r{A}{G}
+%<ger_fut2%>      ! -{E}c{A}{G}
+%<ger_past%>      ! -{D}{I}{G}
 %<ger_abst%>      ! -Uv
 
-%<gpr_fut%>
-%<gpr_past%>
-%<gpr_fut2%>
-%<gpr_pot%>
+! Verbal adjectives
+%<gpr_rsub%>      !
+%<gpr_fut%>       ! -{I}r/-{A}r
+%<gpr_past%>      ! -{G}{A}n
 
+%<gpr_fut2%>      ! -{E}c{A}{G}
+%<gpr_pot%>       ! {j}{I}/-c{I}
+
+! Verbal adverbs
 %<gna_while%>
-%<gna_perf%>
-%<gna_when%>
-%<gna_neg%>
-
-%<prc_perf%>      ! -Ip 
-%<prc_fut%>       ! 
-%<prc_impf%>      ! 
-%<prc_cond%>      ! 
-%<prc_past%>      ! 
-
-%<px1sg%>         !
-%<px2sg%>         !
-%<px3sp%>         !
-%<px1pl%>         !
-%<px2pl%>         !
-
-%<p1%>            !
-%<p2%>            !
-%<p3%>            !
-
-%<sg%>            !
-%<pl%>            ! Plural                      !
-
-%<attr%>          ! Attributive                 !
-%<subst%>         ! Substantive                 !
-%<advl%>          ! 
-
-%<pers%>          ! Personal                    !
-%<dem%>
-%<ref%>
```

Comment:
This tag was removed and should be added back.

Reply:
Sure added back